packages/foreman/katello-selinux/katello-selinux.spec

```diff
@@ -41,7 +41,6 @@ BuildRequires:  /usr/bin/pod2man
 BuildArch:      noarch
 
 Requires:           foreman-selinux
-Requires:           candlepin-selinux >= 3.1.10
```

Comment:
Can `katello-selinux` work without `candlepin-selinux`? I thought there was a SELinux technical reason it was added here.

Reply:
You are correct -- https://github.com/Katello/katello-selinux/pull/22#discussion_r419471890